rsyslog.service.in

```diff
@@ -6,7 +6,9 @@ Documentation=http://www.rsyslog.com/doc/
 
 [Service]
 Type=notify
-ExecStart=@sbindir@/rsyslogd -n
+@syslogd_options_env_file@
+ExecStart=@sbindir@/rsyslogd -n@syslogd_options@
+UMask=0066
```

Comment:
You slipped an unrelated change into this pull request. The UMask change I mean.